x/blob/client/cli/payforblob.go

```diff
@@ -28,69 +30,149 @@ const (
 	// FlagNamespaceVersion allows the user to override the namespace version when
 	// submitting a PayForBlob.
 	FlagNamespaceVersion = "namespace-version"
+
+	// FlagFileInput allows the user to provide file path to the json file
+	// for submitting multiple blobs.
+	FlagFileInput = "input-file"
 )
 
 func CmdPayForBlob() *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "PayForBlobs namespaceID blob",
+		Use: "PayForBlobs [namespaceID blob]",
 		// This example command can be run in a new terminal after running single-node.sh
 		Example: "celestia-appd tx blob PayForBlobs 0x00010203040506070809 0x48656c6c6f2c20576f726c6421 \\\n" +
 			"\t--chain-id private \\\n" +
 			"\t--from validator \\\n" +
 			"\t--keyring-backend test \\\n" +
 			"\t--fees 21000utia \\\n" +
-			"\t--yes",
-		Short: "Pay for a data blob to be published to Celestia.",
-		Long: "Pay for a data blob to be published to Celestia.\n" +
-			"namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n" +
-			"blob must be a hex encoded string of any length.\n" +
-			// TODO: allow for more than one blob to be sumbmitted via the CLI
-			"This command currently only supports a single blob per invocation.\n",
+			"\t--yes \\\n" +
+			"\n" +
+			"celestia-appd tx blob PayForBlobs --input-file path/to/blobs.json \\\n" +
+			"\t--chain-id private \\\n" +
+			"\t--from validator \\\n" +
+			"\t--keyring-backend test \\\n" +
+			"\t--fees 21000utia \\\n" +
+			"\t--yes \\\n" +
+			"\\\n",
```

Comment:
[nit] when the example is printed, it contains trailing `\` after the last `--yes` and it shouldn't. Also there looks like one extra `\` after the second command.

```suggestion
			"\t--yes \n\n" +
			"celestia-appd tx blob PayForBlobs --input-file path/to/blobs.json \\\n" +
			"\t--chain-id private \\\n" +
			"\t--from validator \\\n" +
			"\t--keyring-backend test \\\n" +
			"\t--fees 21000utia \\\n" +
			"\t--yes \n",
```